server/sonar-server/src/main/java/org/sonar/server/user/UserUpdater.java

```diff
@@ -175,45 +178,80 @@ private UserDto createNewUserDto(DbSession dbSession, NewUser newUser) {
     return userDto;
   }
 
-  private void updateUserDto(DbSession dbSession, UpdateUser updateUser, UserDto userDto) {
+  private boolean updateUserDto(DbSession dbSession, UpdateUser updateUser, UserDto userDto) {
     List<Message> messages = newArrayList();
+    boolean changed = updateName(updateUser, userDto, messages);
```

Comment:
- changed is already the first, no ?
- || is not working as as soon as the first one is true, it doesn't execute the other ones.
- I'll try ||=